src/stages/evaluate.ts

```diff
@@ -305,6 +310,7 @@ function isAbilityCompletedByFilledSlotsOnCurrentTurn<T, G>(
 
 function getMissingSlotsOnSlotStatus<T, G>(
   getState: () => WolfState,
+  slots: Slot<G>[],
   abilities: Ability<T, G>[],
```

Comment:
Parameter `slots` and `abilities` can be simplified to pass in `flow` which contains both. Simplifies parameter from 2 -> 1